src/services/AddressService.test.js

```diff
@@ -0,0 +1,58 @@
+import { getFullAddress } from './AddressService';
+
+describe('parseAddress', () => {
+    it('should return a friendly message when the address is empty', () => {
+        //arrange
+        const emptyAddress = {};
+
+        //act
+        const actual = getFullAddress(emptyAddress);
+
+        //assert
+        expect(actual).toEqual(
+            '1745 T Street Southeast1, Louisville, KY  40219'
+        );
+    });
+
+    it('should return a friendly message when the address is matched', () => {
+        //arrange
+        const emptyAddress = [
```

Comment:
the variable name to me suggests that the address would be empty, but you have a valid address?